int8 ring_id
---
rmi/PoseCoords pose